sdk/include/psdk/shobjidl.idl

```diff
@@ -2196,6 +2196,158 @@ interface IDefaultExtractIconInit : IUnknown
                        [in] int iIcon);
 }
 
+
+cpp_quote("#if (NTDDI_VERSION >= NTDDI_LONGHORN)")
```

Comment:
Hmm I don't see a similar guard in MS's file. There are NTDDI_VISTA checks for the surrounding interfaces but not these.

Reply:
yeah but those interfaces are supposed to be Vista and up, hence why I added the guard

Reply:
I don't have a strong preference either way.